topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md

```diff
@@ -0,0 +1,160 @@
+
+> After successfully logging in to the Galaxy server, Go to the [NCBI-Blast+](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/2.10.1+galaxy2) tool.  
+> Since MAdLandDB is the collection of protein sequences, You can perform [BLASTp](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/2.10.1+galaxy2) and [BLASTx](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastx_wrapper/2.10.1+galaxy2) tools.
+
+>
+> <hands-on-title> Similarity search against MAdLand Database </hands-on-title>
+>
+> 1. {% icon tool %} As input file, select the protein sequences 
+> 2. Choose the protein BLAST database *MAdLandDB* with the following parameters:
+>    - In *"input Options"*:
+>        - {% icon param-file %} *"Protein query sequence(s)"*: `output` (Input dataset)
+>        - *"Subject database/sequences"*: `Locally installed BLAST database`
+>        - *"Protein BLAST database"*: `MAdLandDB`
+>    - In *"Output Options"*:
+>       - *"Output format"*: `Tabular (extended 25 columns)` (optional)
+
+> <img src="../../images/ncbi-blast-against-the-madland/blast-example.png" alt="blast against madland" width="80%">
+
+# Blast output 
+
+>{% icon tool %} The following 12 columns are predicted in the tabular formatted output file of the Blast result (you can select desired output format options):  
+
+<table>
+  <tr>
+    <th>Column</th>
+    <th>NCBI name</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>1</td>
+    <td>qseqid</td>
+    <td>Query Seq-id (ID of your sequence)</td>
+  </tr>
+  <tr>
+    <td>2</td>
+    <td>sseqid</td>
+    <td>Subject Seq-id (ID of the database hit)</td>
+  </tr>
+   <tr>
+    <td>3</td>
+    <td>pident</td>
+    <td>Percentage of identical matches</td>
+  </tr>
+   <tr>
+    <td>4</td>
+    <td>length</td>
+    <td>Alignment length</td>
+  </tr>
+   <tr>
+    <td>5</td>
+    <td>mismatch</td>
+    <td>Number of mismatches</td>
+  </tr>
+   <tr>
+    <td>6</td>
+    <td>gapopen</td>
+    <td>Number of gap openings</td>
+  </tr>
+   <tr>
+    <td>7</td>
+    <td>qstart</td>
+    <td>Start of alignment in query</td>
+  </tr>
+   <tr>
+    <td>8</td>
+    <td>qend</td>
+    <td>End of alignment in query</td>
+  </tr>
+   <tr>
+    <td>9</td>
+    <td>sstart</td>
+    <td>Start of alignment in subject (database hit)</td>
+  </tr>
+   <tr>
+    <td>10</td>
+    <td>send</td>
+    <td>End of alignment in subject (database hit)</td>
+  </tr>
+    <tr>
+    <td>11</td>
+    <td>evalue</td>
+    <td>Expectation value (E-value)</td>
+  </tr>
+    <tr>
+    <td>12</td>
+    <td>bitscore</td>
+    <td>Bit score</td>
+  </tr>
+</table>
+
+For more details for BLAST analysis and output, we recommand you to follow the [Similarity-searches-blast](https://training.galaxyproject.org/training-material/topics/genome-annotation/tutorials/genome-annotation/tutorial.html#similarity-searches-blast) tutorial.
```

Comment:
This training is still under development, so perhaps you could avoid to reference it at the moment.